# Task 0

* Посмотрите предков класса `IllegalArgumentException`, `RuntimeException`, `Exception`

IllegalArgumentException -> RuntimeException -> Exception -> Throwable

# Task 1

* Напишите вместо `catch (Exception e)`
  * `catch (PasswordValidationException e)`
  * `catch (EmailValidationException e)`

Сделайте ошибку сначала в email и проверьте оба варианта, а потом в password

Сделал

# Task 2

* По аналогии с классной работой сделайте класс `MathService`, который выполняет деление двух чисел
* Предусмотрите генерацию собственного исключения `ZeroException`
* Предусмотрите обработку этого исключение в `Main`

Сделиль